test/dotnet-msbuild.Tests/GivenDotnetRestoreInvocation.cs

```diff
@@ -29,6 +29,10 @@ public class GivenDotnetRestoreInvocation
         [InlineData(new string[] { "--no-dependencies" }, "-property:RestoreRecursive=false")]
         [InlineData(new string[] { "-v", "minimal" }, @"-verbosity:minimal")]
         [InlineData(new string[] { "--verbosity", "minimal" }, @"-verbosity:minimal")]
+        [InlineData(new string[] { "--use-lock-file" }, "-property:RestorePackagesWithLockFile=true")]
+        [InlineData(new string[] { "--locked-mode" }, "-property:RestoreLockedMode=true")]
+        [InlineData(new string[] { "--reevaluate" }, "-property:ReevaluateRestoreGraph=true")]
+        [InlineData(new string[] { "--lock-file-path", "<lockFilePath>" }, "-property:NuGetLockFilePath=<lockFilePath>")]
```

Comment:
Upper case, underscores for parameter (see `dotnet add -h`). 

`FILE_PATH

Reply:
this is just a test case with actual value of LOCK_FILE_PATH.